Mobile Test Runner for Jama/Mobile Test Runner for JamaTests/TestCycleListModelUnitTests.swift

```diff
@@ -21,6 +21,7 @@ class TestCycleListModelUnitTests: XCTestCase {
     var projectId = 123
     var cycle1Name = "cycle1"
     var cycle2Name = "cycle2"
+    var testPlanId = 12
     var testCycleItemType = 36
```

Comment:
Is this why the hardcoded 36 above? To differentiate if it is a testCycle or not testCycle? Where did those numbers come from?